bin/p2-rctl/main.go

```diff
@@ -34,57 +34,57 @@ import (
 )
 
 const (
-	CMD_CREATE   = "create"
-	CMD_DELETE   = "delete"
-	CMD_REPLICAS = "set-replicas"
-	CMD_LIST     = "list"
-	CMD_GET      = "get"
-	CMD_ENABLE   = "enable"
-	CMD_DISABLE  = "disable"
-	CMD_ROLL     = "rolling-update"
-	CMD_SCHEDUP  = "schedule-update"
+	cmdCreateText   = "create"
+	cmdDeleteText   = "delete"
+	cmdReplicasText = "set-replicas"
+	cmdListText     = "list"
+	cmdGetText      = "get"
+	cmdEnableText   = "enable"
+	cmdDisableText  = "disable"
+	cmdRollText     = "rolling-update"
+	cmdSchedupText  = "schedule-update"
```

Comment:
Elsewhere, there is already a `cmdSchedup`, and this just provides the text of the command. Mostly I just care about removing CAPS and not having two variables with the same name that vary only in the casing pattern used.